docs/mkdocs.yml

```
@@ -3,8 +3,12 @@ site_name: BigQuery ETL
 site_description: Mozilla BigQuery ETL
 site_author: Mozilla Data Platform Team
 
+# Repository
+repo_name: mozilla/bigquery-etl/
```

Does this need to be set? Looking at the docs mkdocs seems to automatically extract this name from the `repo_url`: https://www.mkdocs.org/user-guide/configuration/#repo_name

---

mkdocs-material's default repo name is just "Github", and I thought it would be nice if we change it to our actual repo name. I don't have a strong opinion about this though, happy to remove it if you don't find it necessary :) https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#repository-name

---

Okay, cool. In that case we should remove the `/` at the end.
```suggestion
repo_name: mozilla/bigquery-etl
```